cmd/draft/history.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"fmt"
+	"github.com/Azure/draft/pkg/draft"
+	"github.com/spf13/cobra"
+	"golang.org/x/net/context"
+	"io"
+)
+
+const historyDesc = `Display the build history of a Draft application.`
+
+type historyCmd struct {
+	client  *draft.Client
+	out     io.Writer
+	appName string
+	max     int64
+	args    []string
+}
+
+func newHistoryCmd(out io.Writer) *cobra.Command {
+	hc := &historyCmd{
+		out:  out,
+		args: []string{"app-name"},
+	}
+	cmd := &cobra.Command{
+		Use:   "history <app-name>",
```

Comment:
Would it make sense to pull the app name from the current environment specified in draft.toml? https://github.com/Azure/draft/pull/558 makes this enhancement. This would be useful for `draft history` so a user can call `draft up` multiple times, then call `draft history` to see the application's history.